src/screens/regionPicker/RegionPicker.tsx

```diff
@@ -0,0 +1,126 @@
+import AbFlag from 'assets/flags/ab-flag.png';
+import BcFlag from 'assets/flags/bc-flag.png';
+import MbFlag from 'assets/flags/mb-flag.png';
+import NbFlag from 'assets/flags/nb-flag.png';
+import NlFlag from 'assets/flags/nl-flag.png';
+import NtFlag from 'assets/flags/nt-flag.png';
+import NsFlag from 'assets/flags/ns-flag.png';
+import NuFlag from 'assets/flags/nu-flag.png';
+import OnFlag from 'assets/flags/on-flag.png';
+import PeFlag from 'assets/flags/pe-flag.png';
+import QcFlag from 'assets/flags/qc-flag.png';
+import SkFlag from 'assets/flags/sk-flag.png';
+import YtFlag from 'assets/flags/yt-flag.png';
+import {Region} from 'shared/Region';
+
+interface RegionItemProps {
+  code: Region;
+  flagIcon: any;
+  name: string;
+  selected: boolean;
+  onPress: (code: Region) => void;
+}
+
+const items: Omit<RegionItemProps, 'onPress' | 'selected'>[] = [
+  {code: 'AB', flagIcon: AbFlag, name: 'Alberta'},
+  {code: 'BC', flagIcon: BcFlag, name: 'British Columbia'},
+  {code: 'MB', flagIcon: MbFlag, name: 'Manitoba'},
+  {code: 'NB', flagIcon: NbFlag, name: 'New Brunswick'},
+  {code: 'NL', flagIcon: NlFlag, name: 'Newfoundland and Labrador'},
+  {code: 'NT', flagIcon: NtFlag, name: 'Northwest Territories'},
+  {code: 'NS', flagIcon: NsFlag, name: 'Nova Scotia'},
+  {code: 'NU', flagIcon: NuFlag, name: 'Nunavut'},
+  {code: 'ON', flagIcon: OnFlag, name: 'Ontario'},
+  {code: 'PE', flagIcon: PeFlag, name: 'Prince Edward Island'},
+  {code: 'QC', flagIcon: QcFlag, name: 'Quebec'},
+  {code: 'SK', flagIcon: SkFlag, name: 'Saskatchewan'},
+  {code: 'YT', flagIcon: YtFlag, name: 'Yukon'},
+];
+
+const RegionItem_ = ({code, onPress, name, flagIcon, selected}: RegionItemProps) => (
+  <>
+    <TouchableOpacity onPress={() => onPress(code)}>
+      <Box paddingVertical="s" flexDirection="row" alignContent="center" justifyContent="space-between">
+        <Box flexDirection="row" alignItems="center" paddingVertical="s">
+          <Image source={flagIcon} style={styles.flag} />
+          <Text variant="bodyText" color="overlayBodyText" marginHorizontal="m">
+            {name}
+          </Text>
+        </Box>
+        <Box alignSelf="center">{selected && <Icon size={32} name="icon-check" />}</Box>
+      </Box>
+    </TouchableOpacity>
+    <Box height={1} marginHorizontal="-m" backgroundColor="overlayBackground" />
+  </>
+);
+const RegionItem = React.memo(RegionItem_);
+
+export const RegionPickerScreen = () => {
+  const [i18n] = useI18n();
+  const {setRegion: persistRegion} = useStorage();
+  const [selectedRegion, setSelectedRegion] = useState<Region>('None');
+  const {navigate} = useNavigation();
+
+  return (
+    <Box flex={1} backgroundColor="overlayBackground">
+      <SafeAreaView style={styles.flex}>
+        <ScrollView style={styles.flex}>
+          <Box flex={1} paddingHorizontal="m">
+            <Text variant="bodySubTitle" color="overlayBodyText" textAlign="center">
+              Select your province or territory
```

Comment:
Was having trouble getting translations to work when adding them to `en.json`. Kept getting "translation not found" errors despite restarting metro/refreshing

Reply:
did you run `yarn generate-translations` after adding them to en.json?

Reply:
That's what I was missing. Thank you!